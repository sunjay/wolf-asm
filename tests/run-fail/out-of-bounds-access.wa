# This program loads a memory address that is beyond the size of the
# memory in the virtual machine

section .code

mov $1, 1024
mov $2, 4
mov $3, 100

mulu $1, $2
add $1, $3

loadu8 $1, $1
