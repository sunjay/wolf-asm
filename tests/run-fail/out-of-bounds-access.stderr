Error: Failed to execute instruction at `0x28`

Caused by:
    Invalid memory access: attempt to access `0x1064` when address must be less than `0x1000`
